src/MCPServer/lib/settings/common.py

```diff
@@ -168,3 +172,5 @@
 LIMIT_GEARMAN_CONNS = config.get('limit_gearman_conns')
 RESERVED_AS_TASK_PROCESSING_THREADS = config.get('reserved_as_task_processing_threads')
 SEARCH_ENABLED = config.get('search_enabled')
+STORAGE_SERVICE_CLIENT_TIMEOUT = config.get('storage_service_client_timeout')
```

Comment:
Yes, it's an implicit requirement when you import storageService because that module looks up `django_settings`. I realize it's not great, I will think about it.